### ISSUE TITLE
Fix make_release script

### ISSUE THREAD

**Issue by oy** [teeworlds/teeworlds#2569]
to use the map versions within teeworlds master branch

**Comment by heinrich5991** [comment]
That wouldn't work for rebuilding old releases.

**Comment by oy** [comment]
Then you have to git version of the actual release or pass the correct map version as parameter.

**Comment by heinrich5991** [comment]
But we do source releases as well. You would no longer be able to reproduce our other release tarballs from there. I think it would be better to integrate the map versions subrepository by SHA.

**Comment by oy** [comment]
Yeah, that's what i was thinking about. Like it's done here https://github.com/teeworlds/teeworlds/blob/master/scripts/download.py#L19
